README.md

```diff
@@ -3,27 +3,39 @@ An app to record what books are in a library and who borrows them. It allows to
 
```

Comment:
- [ ] None of your tests run at all. This is because your tests file names have `.spec.rb` instead of `_spec.rb`. Kindly rename your files. For example this file:

  ```
  base_decorator.spec.rb
  ```
  should be:

  ```
  base_decorator_spec.rb
  ```

  ![eyrur](https://user-images.githubusercontent.com/54848363/198553814-cb48bdcb-4a15-4c75-aebe-f801a8e8756d.jpg)

Reply:
Hi @DammyShittu, Thank you so much for taking the time to leave this excellent review. We missed that and tested a file by, for example, running `rspec spec/students.spec.rb` to test `students class` and that worked.

Fixing that now and we really appreciate your input to make our code better. Cheers!